bears/c_languages/ArtisticStyleBear.py

```diff
@@ -0,0 +1,865 @@
+            For example: If set to ``True``, prefer this::
+
+                if (isFoo)
+                    isFoo = false;
+
+            over this::
+
+                if (isFoo)
+                {
+                    isFoo = false;
+                }
+
+        :param remove_comment_prefix:
+            Remove the preceding '*' in a multi-line comment that begins a
+            line. A trailing '*', if present, is also removed. Text that is
+            less than one indent is indented to one indent. Text greater than
+            one indent is not changed. Multi-line comments that begin a line,
+            but without the preceding '*', are indented to one indent for
+            consistency. This can slightly modify the indentation of commented
+            out blocks of code. Lines containing all '*' are left unchanged.
+            Extra spacing is removed from the comment close '*/'.
+            For example: If set to ``True``, prefer this::
+
+                /*
+                    comment line 1
+                    comment line 2
+                */
+
+            over this::
+
+                /*
+                 * comment line 1
+                 * comment line 2
+                 */
+
+        """
+        rules_map = {
+            '--attach-namespaces': attach_braces_to_namespace,
+            '--attach-classes': attach_braces_to_class,
+            '--attach-inlines': attach_braces_to_inline,
+            '--attach-extern-c': attach_braces_to_extern,
+            '--indent-classes': allow_indent_classes,
+            '--indent-modifiers': allow_indent_modifiers,
+            '--indent-switches': allow_indent_switches,
+            '--indent-cases': allow_indent_cases,
+            '--indent-namespaces': allow_indent_namespaces,
+            '--indent-labels': allow_indent_labels,
+            '--indent-preproc-block': allow_indent_preproc_block,
+            '--indent-preproc-define': allow_indent_preproc_definition,
+            '--indent-preproc-cond': allow_indent_preproc_conditionals,
+            '--indent-col1-comments': allow_indent_column_one_comments,
+            '--break-blocks': allow_pad_header_blocks,
+            '--pad-oper': allow_pad_operators,
+            '--pad-paren': allow_pad_parenthesis,
+            '--pad-paren-out': allow_pad_parenthesis_out,
+            '--pad-paren-in': allow_pad_parenthesis_in,
+            '--delete-empty-lines': delete_empty_lines_in_func,
+            '--break-closing-brackets': break_closing_braces,
+            '--break-elseifs': break_elseifs,
+            '--break-one-line-headers': break_one_line_headers,
+            '--add-brackets': add_braces_to_one_line_conditionals,
+            '--remove-brackets': remove_braces_from_one_line_conditionals,
+            '--remove-comment-prefix': remove_comment_prefix
+        }
+        args = ['--suffix=none', '--dry-run']
+        if bracket_style:
+            args += (''.join(('--style=', bracket_style)),)
```

Comment:
and I don't think you need to use `join` for just two values, I think this is easier and better to read ;)
```
args.append('--style=' + bracket_style)
```